sipa/utils/__init__.py

```diff
@@ -82,6 +87,34 @@ def support_hotline_available():
         return False
 
 
+def make_link(text):
+    pat_url = re.compile(  r'''(?x)((http|https)://(\S+))''')
+
+    for url in re.findall(pat_url, text):
+       text = text.replace(url[0], '<a href="%(url)s">%(url)s</a>' % {"url" : url[0]})
+
+    return text
+
+def meetingcal():
+    url = "https://agdsn.de/cloud/remote.php/dav/public-calendars/bgiQmBstmfzRdMeH?export"
+
+    calendar = icalendar.Calendar.from_ical(requests.get(url).text)
+    events = recurring_ical_events.of(calendar).between(datetime.now(), datetime.now() + relativedelta(months=1))
+
+    next_meetings = []
+    for event in events:
+        next_meetings.append({
+            "title": event["SUMMARY"],
+            "datetime": event["DTSTART"].dt,
+            "location": event["LOCATION"] if "LOCATION" in event else "-",
+            "location_link": markdown.markdown(event["LOCATION"]) if "LOCATION" in event else "-"
+        })
+
+    next_meetings = sorted(next_meetings, key=lambda x: x["datetime"])
+    next_meetings = next_meetings[:4]
```

Comment:
It would be better if only a few events (e.g. 3 or 4) are shown in the sidebar, but more events shown on the dedicated page.